scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -380,6 +408,8 @@ def validate_assets(input_dirs: List[Path],
                         help="Comma-separated list of changed files, used to filter assets")
     parser.add_argument("-n", "--check-names", action="store_true",
                         help="Check asset names")
+    parser.add_argument("-d", "--check-dockerfile", action="store_true",

```

Comment:
Let's just always check the Dockerfile and not add another arg. I can't think of a situation where we wouldn't want to.